lib/autocomplete_source.rb

```diff
@@ -82,7 +82,7 @@ def get_search_regex(search_param)
     # and returns an array of stripped and lowercase words for actual searching or use in keys
     def get_search_terms(search_term)
       terms = search_term.is_a?(Array) ? search_term.map {|term| term.split(',')}.flatten : (search_term.blank? ? [] : search_term.split(','))
-      terms.map {|term| term.strip.downcase}
+      terms.map { |term| term.strip.downcase }
```

Comment:
I originally tried changing this one, but it actually introduced a bit of a bug (that's what the test about Bear and Beatrice and Østenfor sol og vestenfor måne was for). It uses ruby's built-in downcase because the tags-in-fandom autocomplete code uses ruby's built-in downcase (e.g. Tag.add_to_autocomplete and TagSetAssociation.add_to_autocomplete). The javascript code doesn't do any downcasing there.

But if you'd like, I can change all downcasing in the autocomplete code to use mb_chars.

Reply:
Ugh, that sounds like more trouble than it's worth at this point and not necessary to fix this issue, so let's leave that alone for now! Thanks.